deployments/helm/stg/us_east_2/versions.tf

```diff
@@ -0,0 +1,20 @@
+# Iteration: 1625064074
+
+terraform {
+  required_version = "~> 0.15.1"
+  required_providers {
+    aws = {
+      version = "~> 3.46.0"
+    }
+
+    kustomization = {
+      source  = "kbst/kustomize"
+      version = "0.2.0-beta.3"
```

Comment:
version = "0.7.0"

Reply:
`│ Could not retrieve the list of available versions for provider kbst/kustomize: no available releases match the given constraints 0.7.0`